### DISCUSSION TITLE
Warnings. None of paired reads aligned properly.

### DISCUSSION THREAD

**Discussion by DanieleDeco** [ablab/spades#1021] [Q&A]
### Description of bug

I run Spades for a few metagenomic samples and in a few of them, I got the following warnings.
=== Error correction and assembling warnings:
 * 2:25:51.222  6985M / 14G   WARN    General                 (pair_info_count.cpp       : 358)   Unable to estimate insert size for paired library #0
 * 2:25:51.247  6985M / 14G   WARN    General                 (pair_info_count.cpp       : 364)   None of paired reads aligned properly. Please, check orientation of your read pairs.
 * 2:25:51.278  6985M / 14G   WARN    General                 (repeat_resolving.cpp      :  81)   Insert size was not estimated for any of the paired libraries, repeat resolution module will not run.
 * 4:11:14.894  6192M / 14G   WARN    General                 (pair_info_count.cpp       : 187)   Single reads are not used in metagenomic mode

Do you know what the problem could be?

The paired reads are pre-processed with trimmomatric and checked with fastq-pair.

Thanks

Daniele

[warnings.log](https://github.com/ablab/spades/files/9487351/warnings.log)
[params.txt](https://github.com/ablab/spades/files/9487352/params.txt)
[spades.log](https://github.com/ablab/spades/files/9487353/spades.log)

### spades.log

[spades.log](https://github.com/ablab/spades/files/9487369/spades.log)

### params.txt

[params.txt](https://github.com/ablab/spades/files/9487370/params.txt)

### SPAdes version

Spades 15.2

### Operating System

Debian 5.9.15

### Python Version

Python

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
Your input reads are corrupted. E.g. the left ones do not correspond to the right ones. Ensure you're using paired-end aware trimmer.

**Reply by DanieleDeco** [reply]
I checked that with fastq-pair and It seems that all the reads are paired.

**Reply by asl** [reply]
fastq-pair only checks read names. So, if your reads were screwed in other way, then it will not help. Alternatively, your assembly is so fragmented that there are no long contigs usable for estimation of fragment length distribution.
